default_app/default_app.js

```diff
@@ -20,5 +20,6 @@ exports.load = function (appUrl) {
     })
     mainWindow.loadURL(appUrl)
     mainWindow.focus()
+    mainWindow.webContents.getResourceUsage((x) => console.log(x));
```

Comment:
This call seems to always fail with "Insufficient number of arguments" - any idea what I'm doing wrong with marshaling this @zcbenz?